### PR TITLE
Implements actual invisibility

### PR DESCRIPTION
**Pull request by ike709**
https://streamable.com/8zbc1n

### CODE REVIEW

File: OpenDreamShared/Net/Packets/PacketDeltaGameState.cs [hunk 1]
```diff
@@ -272,6 +275,11 @@ private enum ClientValueID {
                     stream.WriteByte((byte)ClientDelta.NewPerspective.Value);
                 }
 
+                if (newSeeInvisible)
+                {
+                    stream.WriteByte(ClientDelta.NewSeeInvisible.Value);
```
Comment (wixoaGit) [inline OpenDreamShared/Net/Packets/PacketDeltaGameState.cs:280]:
```suggestion
                    stream.WriteByte((byte)ClientValueID.SeeInvisible);
                    stream.WriteByte(ClientDelta.NewSeeInvisible.Value);
```

**Review by wixoaGit — CHANGES_REQUESTED** [review]
`SeeInvisible` needs to be serialized in `PacketFullGameState`'s `ReadClientSection` and `WriteClientSection`

**Comment by ike709** [comment]
Done.

File: OpenDreamShared/Net/Packets/PacketDeltaGameState.cs [hunk 2]
```diff
@@ -272,6 +275,11 @@ private enum ClientValueID {
                     stream.WriteByte((byte)ClientDelta.NewPerspective.Value);
                 }
 
+                if (newSeeInvisible)
+                {
+                    stream.WriteByte((byte)ClientDelta.NewSeeInvisible.Value);
```
Comment (wixoaGit) [inline OpenDreamShared/Net/Packets/PacketDeltaGameState.cs:280]: You have to write which value this is before actually writing the value
```suggestion
                    stream.WriteByte((byte)ClientValueID.SeeInvisible);
                    stream.WriteByte((byte)ClientDelta.NewSeeInvisible.Value);
```

**Review by wixoaGit — CHANGES_REQUESTED** [review]

**Comment by ike709** [comment]
There

**Review by wixoaGit — APPROVED** [review]

**Pull request merged**